app/models/manageiq/providers/azure/refresh_helper_methods.rb

```diff
@@ -64,6 +64,11 @@ def resource_groups
     @resource_groups ||= @rgs.list
   end
 
+  def resources_for(resource_type)
+    filter = "resourceType eq '#{resource_type}'"
+    @res.list_all(:filter => filter)
+  end
+
```

Comment:
is this method used?

Reply:
Oh, I thought I removed it. The original plan was to use the ResourceService, until I discovered that it did not include the information that I needed.